geo-types/src/lib.rs

```diff
@@ -1,23 +1,39 @@
 #![warn(missing_debug_implementations)]
 #![doc(html_logo_url = "https://raw.githubusercontent.com/georust/meta/master/logo/logo.png")]
-//! The `geo-types` library provides geospatial primitive types and traits to the [`GeoRust`](https://github.com/georust)
-//! crate ecosystem.
+//! The `geo-types` library provides geospatial primitive types for the [GeoRust] ecosystem.
 //!
-//! In most cases, you will only need to use this crate if you're a crate author and want compatibility
-//! with other `GeoRust` crates. Otherwise, the [`geo`](https://crates.io/crates/geo) crate re-exports these types and
-//! provides geospatial algorithms, while the [`geojson`](https://crates.io/crates/geojson) crate allows serialising
-//! and de-serialising `geo-types` primitives to GeoJSON.
+//! In most cases, you will only need to use this crate if you’re a crate author and want
+//! compatibility with other GeoRust crates. Otherwise, the [`geo`](https://crates.io/crates/geo)
+//! crate re-exports these types and provides geospatial algorithms.
+//!
+//! # Types
+//!
+//! - **[`Coordinate`]**: A two-dimensional coordinate all other geometry types are based on
```

Comment:
The distinction between Coordinate and Points is always little tricky. Do you think it would help to clarify by *avoiding* calling it a geometry type?

Maybe something like:

**[`Coordinate`]**: A two-dimensional coordinate. All Geometry types are composed of Coordinates, though Coordinate itself is not a Geometry type.
**[`Point`]**: ...

I'm not sure if the verbosity is worth it, so feel free to ignore this suggestion.